### PR TITLE
Jen wordlists

### PR DESCRIPTION
**Pull request by Kristina-Pianykh**
The issue with the lists is [here](https://github.com/lexibank/lexibank/issues/173#issuecomment-541573271).

I had to change the author's name from Kleinewillingh**ö**fer to Kleinewillingh**oe**fer because oddly with the original spelling I got an error that the conceptlist ID is invalid.

Concerning the changes in Dolch-1936-220, I've spotted that in the pronominal system there is a three-way distinction between subject, object and genetive case. It is distinct in the male person pronoun (he-him-his), but not in the female person pronoun (she-her-her) where the object and genetive cases are represented by one form. So I left _her_ unmapped.

_him_ and _them_ were also left unlinked since there is no concept OBLIQUE CASE for either of them (compare with OBLIQUE CASE OF WE, YOU and THEE).

### CODE REVIEW

File: concepticondata/references/references.bib [hunk 1]
```diff
@@ -2316,3 +2316,25 @@ @incollection{Lithgow1976
   Year                                = {1976},
   Url                                 = {http://sealang.net/archives/pl/pdf/PL-C39.441.pdf}
 }
+
+@Misc{Kleinewillinghoefer2015,
+  Author                              = {Kleinewillinghoefer, Ulrich},
```
Comment (chrzyki) [inline concepticondata/references/references.bib]: Name with Umlauts here?

File: concepticondata/references/references.bib [hunk 2]
```diff
@@ -2316,3 +2316,25 @@ @incollection{Lithgow1976
...
+  Title                               = {Bikwin-Jen - Comparative Wordlist (Swadesh 100)},
+  Url                                 = {https://www.blogs.uni-mainz.de/fb07-adamawa/files/2011/11/Bikwin-Jen-comparative-wordlist-100.pdf},
+  Year                                = {2015}
+}
+
+@Misc{Kleinewillinghoefer2014,
+  Author                              = {Kleinewillinghoefer, Ulrich},
```
Comment (chrzyki) [inline concepticondata/references/references.bib]: Likewise?

**Comment by chrzyki** [comment]
Looks great & fine by me, many thanks!

**Comment by LinguList** [comment]
Yes, names have the problem that they need to be alphanumeric, this is like in programming languages: only alphabet plus numbers, no Umlaut.

**Comment by LinguList** [comment]
Thanks!

**Pull request merged**